Curve-finance-suite/src/agents/curve.dao.create.lockevent.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  Finding,
+  HandleTransaction,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+} from 'forta-agent';
+
+import Web3 from 'web3';
+import abi from '../utils/curve.dao.abi';
+
+// @ts-ignore
+import abiDecoder from 'abi-decoder';
+abiDecoder.addABI(abi);
+
+export const web3 = new Web3();
+
+export const createLock = {
+  name: 'create_lock',
+  outputs: [],
+  inputs: [
+    { type: 'uint256', name: '_value' },
+    { type: 'uint256', name: '_unlock_time' },
+  ],
+  stateMutability: 'nonpayable',
+  type: 'function',
+  gas: 74281465,
+};
+
+const createFinding = (alertId: string): Finding => {
+  return Finding.fromObject({
+    name: 'Create Lock Event called',
+    description: 'Create Lock Event funciton called on pool',
```

Comment:
on what pool? Is the address field in metadata the pool? Can you add the pool in the metadata? Can you add any metadata to make this finding more informative?